### PR TITLE
[MDCT-2920] [MDCT-2922] Section F & G

### PR DESCRIPTION
**Pull request by BearHanded**
### Description

Section 3 - F & G

### Related ticket(s)

MDCT-2920
MDCT-2922

---
### How to test

Compare the diff to the changes requested in the ticket, proofread, then spin it up.

### Important updates

---
### Author checklist

- [x] I have performed a self-review of my code
- [ ] I have added [thorough](https://bit.ly/3zPrxuZ) tests, if necessary
- [ ] I have updated relevant documentation, if necessary
---

_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_

### CODE REVIEW

File: services/database/data/seed/seed-section-base-2023.json [hunk 1]
```diff
@@ -8323,7 +8323,7 @@
                             "id": "2023-03-g-01-07-b",
                             "hint": "This is the total number for all children between 0-18 years from question 1.",
                             "type": "integer",
-                            "label": "How many children were enrolled in Separate CHIP for at least 90 continuous days during FFY 2022?",
+                            "label": "How many children were enrolled in Separate CHIP for at least 90 continuous days during FFY 2023?",
                             "answer": {
```
Comment (britt-mo) [inline services/database/data/seed/seed-section-base-2023.json:8327]: lowercase `s` here

File: services/database/data/seed/seed-section-base-2023.json [hunk 2]
```diff
@@ -8323,7 +8323,7 @@
                             "id": "2023-03-g-01-07-b",
                             "hint": "This is the total number for all children between 0-18 years from question 1.",
                             "type": "integer",
-                            "label": "How many children were enrolled in Separate CHIP for at least 90 continuous days during FFY 2022?",
+                            "label": "How many children were enrolled in Separate CHIP for at least 90 continuous days during FFY 2023?",
```
Comment (britt-mo) [inline services/database/data/seed/seed-section-base-2023.json:8326]: I think this is a lowercase `s`?

Comment (BearHanded) [reply services/database/data/seed/seed-section-base-2023.json:8326]: good catch, thank you

File: services/database/data/seed/seed-section-base-2023.json [hunk 3]
```diff
@@ -7777,13 +7777,13 @@
             "parts": [
               {
                 "id": "2023-03-g-01",
-                "text": "Tell us about the children receiving dental benefits in your Separate CHIP program. Include children who are receiving full benefits and those who are only receiving supplemental dental benefits. Include the unduplicated number of children enrolled in all types of delivery systems (Managed Care, PCCM, and Fee for Service).",
+                "text": "Tell us about the children receiving dental benefits in your Separate CHIP program. Include children who are receiving full benefits and those who are only receiving supplemental dental benefits. Include the unduplicated number of children enrolled in all types of delivery systems (managed care, PCCM, and fee-for-service).",
```
Comment (gmrabian) [inline services/database/data/seed/seed-section-base-2023.json:7780]: lowercase "separate"

**Review by gmrabian — CHANGES_REQUESTED** [review]
some notes to find out from product

File: services/database/data/seed/seed-section-base-2023.json [hunk 4]
```diff
@@ -7610,15 +7610,15 @@
...
+                        "label": "How many cases related to provider billing were investigated in FFY 2023?",
                         "answer": {
                           "entry": null
                         }
                       },
                       {
                         "id": "2023-03-f-01-11",
                         "type": "integer",
-                        "label": "How many cases were referred to appropriate law enforcement officials in FFY 2022?",
+                        "label": "How many cases were referred to appropriate law enforcement officials in FFY 2023?",
```
[the remark this answers is not in the document]
Comment (BearHanded) [reply services/database/data/seed/seed-section-base-2023.json:7621]: Clarified, this needs to be as shown in the ticket text.

File: services/database/data/seed/seed-section-base-2023.json [hunk 5]
```diff
@@ -7547,7 +7547,7 @@
                       {
                         "id": "2023-03-f-01-05-a",
                         "type": "text_multiline",
-                        "label": "What safeguards and procedures do the Managed Care plans have in place?",
+                        "label": "What safeguards and procedures do the managed care plans have in place?",
```
[the remark this answers is not in the document]
Comment (BearHanded) [reply services/database/data/seed/seed-section-base-2023.json:7550]: Text removed in ticket. Field should remain a text field

**Review by benmartin-coforma — APPROVED** [review]
I haven't spun this up, but the code seems to cover all of the requested changes.

**Review by gmrabian — APPROVED** [review]

**Pull request merged**